app/models/stop.rb

```diff
@@ -0,0 +1,5 @@
+class Stop < ApplicationRecord
```

Comment:
You need to create a test for this new model

Reply:
test created